### PR TITLE
Updated landing page 

### PR DESCRIPTION
**Pull request by georgically**
# Landing page updated with contents 
- get tickets button
- brief outline of event 
- keynote speakers section
- sponsors section.

### Note:  
menu navbar and footer are in separate PR

### CODE REVIEW

File: pelicanconf.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 AUTHOR = 'Tech WG'
 SITENAME = 'PyCon Thailand 2020'
-SITEURL = 'https://th.pycon.org/'
+SITEURL = ''
```
Comment (jean) [inline pelicanconf.py]: Don't remove this.

File: theme/static/css/elegant.css [hunk 2]
```diff
@@ -40,7 +40,7 @@ h3,
 h4,
 h5,
 h6 {
-  font-family: Baskerville, Garamond, Georgia, "DejaVu Serif", "Times New Roman",
+  font-family: "Chakra Petch", Garamond, Georgia, "DejaVu Serif", "Times New Roman",
```
Comment (jean) [inline theme/static/css/elegant.css]: `custom.css`, https://elegant.oncrashreboot.com/customize-style but OK ...

File: pelicanconf.py (hunk 1 above)
Comment (georgically) [reply pelicanconf.py]: I cannot review locally when I have this on apparently :(

**Comment by georgically** [comment]
> Looks OK, though spacing on https://deploy-preview-25--lucid-hoover-526812.netlify.com/ looks kinda weird to me.
> No top margin above ">>> print", and
> "Workshop/Conference/..." icons cut off in the middle.
> 
> On my phone the "Workshops" and "Conference" text crash into each other (overlap), and the keynote bios are cut off on both sides.
> If that's OK with you, we can merge ..

I have the same issue too. It's not ok at all. Will fix that in the evening

File: pelicanconf.py (hunk 1 above)
Comment (jean) [reply pelicanconf.py]: If you have to, change it when you're working locally, but don't commit the change. 
It's easier to just work around the broken links --- i.e. if you follow a link to https://th.pycon.org/pages/something , just change it to https://localhost:8000/pages/something so see the local version. 
Where do you run into broken links? I don't see any.
It looks like it's only used for feeds?
https://docs.getpelican.com/en/stable/settings.html#SITEURL

Comment (georgically) [reply pelicanconf.py]: I tried previously adding it and it broke all links in the local version. Removing it seems to work otherwise.

**Comment by jean** [comment]
@Macavirus ping me and/or @bluenex for a review when ready OK?

**Comment by bluenex** [comment]
I think a keynote speaker card is way too big in large screen.

- 1024px

image

- 1440px +

image

**Comment by georgically** [comment]
> I think a keynote speaker card is way too big in large screen.
> 
> * 1024px
> 
> image
> 
> * 1440px +
> 
> image

Hmm... It wasn't this size when I set it? checking it out now

File: theme/static/css/landing_page.css [hunk 3]
```diff
@@ -259,10 +259,6 @@ figure.keynote {
   position: relative;
   overflow: hidden;
   margin: 10px;
-  min-width: 500px;
-  max-width: 600px;
-  min-height: 300px;
-  max-height: 800px;
```
Comment (georgically) [inline theme/static/css/landing_page.css]: Removing the width and height limit stretches the speaker cards to the maximum width on large screen which is way too big

**Comment by georgically** [comment]
@jean I've merged this branch since there's no issues to the PR

**Pull request merged**